gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -175,6 +195,70 @@ def main(args):
                 ).write(training_resources[region].path, overwrite=overwrite)
             logger.info("Done with creating training dataset.")
 
+        # Build plateau and coverage models for autosomes/pseudoautosomal regions,
+        # chromosome X, and chromosome Y
+        if args.build_models:
+            # Check if the training datasets exist.
+            check_resource_existence(
+                "--create-training-set",
+                "--build_models",
+                training_resources.values(),
+                models.values(),
+                overwrite=overwrite,
+            )
+            # Build plateau and coverage models.
+            for region in GENOMIC_REGIONS:
+                logger.info("Building %s plateau and coverage models...", region)
+
+                coverage_model, plateau_models = build_models(
+                    training_resources[region].ht(),
+                    weighted=use_weights,
+                    pops=POPS if use_pops else (),
+                    build_coverage_model=True,
+                )
+                hl.experimental.write_expression(
+                    plateau_models, models[(region, "plateau")]
+                )
+                hl.experimental.write_expression(
+                    coverage_model, models[(region, "coverage")]
+                )
+                logger.info("Done building %s plateau and coverage models.", region)
+
+        # Apply coverage and plateau models to compute expected variant counts and
+        # observed:expected ratio
+        if args.apply_models:
+            logger.info(
+                "Applying %s plateau and coverage models and compute expected variant"
+                " count and observed:expected ratio...",
+                region,
+            )
+
+            # Check if the input/output resources exist.
+            check_resource_existence(
+                "--build_models",
+                "--apply_models",
+                models.values(),
+                testing_resources.values(),
+                overwrite,
+            )
+            # Apply coverage and plateau models for sites on autosomes/pseudoautosomal
+            # regions, chromosome X, and chromosome Y.
+            for region in GENOMIC_REGIONS:
+                apply_models(
+                    preprocess_resources[(region, "exomes")].ht(),
+                    preprocess_resources[(region, "context")].ht(),
+                    mutation_ht,
+                    hl.experimental.read_expression(models[(region, "plateau")]),
+                    hl.experimental.read_expression(models[(region, "coverage")]),
+                    max_af=max_af,
+                    pops=POPS if use_pops else (),
+                    partition_hint=partition_hint * 20,
```

Comment:
I'm not sure if it's the best way to do it. There are three different partition hint ([1](https://github.com/broadinstitute/gnomad-constraint/blob/3f4ebe90560cf4ddfb753b8dbe1d4e373a01d5eb/gnomad_constraint/utils/constraint.py#L114) [2](https://github.com/broadinstitute/gnomad-constraint/blob/3f4ebe90560cf4ddfb753b8dbe1d4e373a01d5eb/gnomad_constraint/utils/constraint.py#L226) [3](https://github.com/broadinstitute/gnomad-constraint/blob/3f4ebe90560cf4ddfb753b8dbe1d4e373a01d5eb/gnomad_constraint/utils/constraint.py#L425)). Should we made these separate parameters and arguments?